lib/fetch/body.js

```diff
@@ -0,0 +1,277 @@
+'use strict'
+
+const util = require('../core/util')
+const { toWebReadable } = require('./util')
+const { kState } = require('./symbols')
+const { Blob } = require('buffer')
+const { Readable } = require('stream')
+const { NotSupportedError } = require('../core/errors')
+const { ReadableStream } = require('stream/web')
```

Comment:
So this is not usable in older versions of Node. I think this is a breaking change because undici was supporting Node 12 before.

Reply:
It still works on node 12.

Reply:
It imports `stream/web` and it fails. Are you sure because `stream/web` isn't available in <16.

Reply:
This file is not imported if version <16

Reply:
Our CI runs node 12,14 and 16

Reply:
Undici used to support the `request.body` from node-fetch before but now it imports something from `stream/web` and this blocks us to update the latest version. I can create a reproduction if you want.

Reply:
Please open an issue and I'll have a look.

Reply:
I think I see the problem. It has nothing to do with node versions. Doesn't work on any version.

Reply:
https://github.com/nodejs/undici/pull/955

Reply:
Thanks! And also there is another issue with the minor versions of Node 16 that doesn't have `stream/web`.

Reply:
New release published with the fixes you wanted.

Reply:
Thank you very much for quick response and fix 🙏🙏